plottable.d.ts

```diff
@@ -1251,7 +1258,7 @@ declare module Plottable {
         class Line extends Drawer {
             constructor(dataset: Dataset);
             protected _applyDefaultAttributes(selection: d3.Selection<any>): void;
-            selectionForIndex(index: number): d3.Selection<void>;
```

Comment:
I would recommend force casting it I think in this case, unless our type here is just wrong